        <html><head></head><body>
        <h1>Report for python-nose-cover3 on toolchain
            <a href="index.html">android</a>
        </h1>
        <p>Return code: 5</p>
        <p>Time to run: 00:00:23</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_android -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data python-nose-cover3
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 462 packages in cache
===> sudo -u tuscan makepkg --nobuild --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: python-nose-cover3 0.1.0-2 (Fri Nov 13 23:10:21 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Packages (8) perl-error-0.17024-1  python-packaging-15.3-2  python2-2.7.10-2  python2-packaging-15.3-2  sqlite-3.9.2-1  git-2.6.2-1  python-setuptools-1:18.5-1  python2-setuptools-1:18.5-1

Total Installed Size:  104.69 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
checking available disk space...
installing python-packaging...
installing python-setuptools...
installing sqlite...
installing python2...
Optional dependencies for python2
    tk: for IDLE
    python2-setuptools [pending]
    python2-pip
installing python2-packaging...
installing python2-setuptools...
installing perl-error...
installing git...
Optional dependencies for git
    tk: gitk and git gui
    perl-libwww: git svn
    perl-term-readkey: git svn
    perl-mime-tools: git send-email
    perl-net-smtp-ssl: git send-email TLS support
    perl-authen-sasl: git send-email TLS support
    python2: various helper scripts [installed]
    subversion: git svn
    cvsps2: git cvsimport
    gnome-keyring: GNOME keyring credential helper
==> Retrieving sources...
  -> Cloning nosecover3 git repo...
Cloning into bare repository '/sources/python-nose-cover3/nosecover3'...
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
  -> Creating working copy of nosecover3 git repo...
Cloning into 'nosecover3'...
done.
Switched to a new branch 'makepkg'
==> Starting prepare()...
==> Sources are ready.
Copied source directory to /sources/python-nose-cover3
===> sudo -u tuscan PATH=/tmp/toolchain/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin CC=arm-linux-androideabi-gcc CXX=arm-linux-androideabi-g++ SYSROOT=/build/android-ndk-r10e/platforms/android-21/arch-arm makepkg --noextract --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: python-nose-cover3 0.1.0-2 (Fri Nov 13 23:10:28 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Entering fakeroot environment...
==> Starting package_python-nose-cover3()...
Traceback (most recent call last):
  File "setup.py", line 9, in <module>
    from ez_setup import use_setuptools
ImportError: No module named ez_setup
==> ERROR: A failure occurred in package_python-nose-cover3().
    Aborting...
===> Printing config.logs
===> Finished printing config.logs></pre></code>
        </body></html>
